libraries/AP_HAL_ChibiOS/hwdef/common/usbcfg.c

```diff
@@ -401,14 +401,14 @@ const USBConfig usbcfg = {
  * Serial over USB driver configuration.
  */
 const SerialUSBConfig serusbcfg1 = {
-#if STM32_OTG2_IS_OTG1
-  &USBD2,
-#else
+#if STM32_USB_USE_OTG1 || STM32_OTG2_IS_OTG1
```

Comment:
This is wrong - you will have broken STM32H730

Reply:
should be fixed, can you check now